src/chainstate/burn/operations/mod.rs

```diff
@@ -415,6 +430,83 @@ impl BlockstackOperationType {
             BlockstackOperationType::DelegateStx(ref mut data) => data.burn_header_hash = hash,
         };
     }
+
+    pub fn pre_stx_to_json(op: &PreStxOp) -> serde_json::Value {
+        json!({
+            "PreStx": {
```

Comment:
Once last nit here: this looks like the first time that casing conventions are breaking from `snake_case` and instead are using `PascalCase`. E.g. `PreStx` vs `pre_stx`

Reply:
I was mimicking what would happen if you were to call the default serialization function (it would use Pascal case because that's how the op structs are named). 

But I have no problem switching to camel case so I'll push that.